master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Installing Calico with Application Layer Policy
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy'
+---
+
+Application Layer Policy for {{site.prodname}} allows you to write policies that enforce against both network layer attributes, like IP addresses or ports and application layer attributes like HTTP Method or Path. It can also enforce policy based on cryptographically secure identities, allowing you to reduce trust in your network.  See [Application Layer Policy Introduction](link me) for more information.
+
+# Kubernetes Cluster Requirements
+
+Application Layer Policy is only supported on Kubernetes, and the cluster must meet the following requirements:
+
+  * Kubernetes v1.9 or later
+  * [Mutating Webhook](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19) support is enabled
+
+# Enabling support in {{site.prodname}}
+
+Support for Application Layer Policy is not enabled by default in {{site.prodname}} installs, since it requires extra CPU and memory resources to operate. Follow the {{site.prodname}} guides for your chosen configuration.  In each guide, in the step where you download and install the `calico.yaml` manifest, modify the URL to replace `calico.yaml` with `calico-app-layer-policy.yaml`, or `canal.yaml` with `canal-app-layer-policy.yaml`. This variant of the manifest will start {{site.prodname}} with Application Layer Policy support enabled.
+
+- [Installing {{site.prodname}} for policy and networking (recommended)](calico)
+
+- [Installing {{site.prodname}} for policy and flannel for networking](flannel)
+
+- [Installing {{site.prodname}} for policy (advanced)](other)
+
+# Install Istio
+
+Application Layer Policy requires you to use Istio in your cluster to function correctly. We support Istio version 0.6.0 or newer.
```

Comment:
This might be something to add to the requirements page, as well. nit: we have been using "later" rather than "newer"